crypto/dh/dh_lib.c

```diff
@@ -78,6 +78,8 @@ static DH *dh_new_intern(ENGINE *engine, OSSL_LIB_CTX *libctx)
     if (ret == NULL)
         return NULL;
 
+    ossl_ffc_params_init(&ret->params);
+
```

Comment:
I'd prefer moving this below - perhaps after the `ret->meth` assignment.

Reply:
Makes sense.

Do you prefer it right after [line 97](https://github.com/openssl/openssl/pull/22185/files/8e0f3474abc5a53bc6304c6b3157a6b09c5cdfac#diff-db882b7e799eaa3257692a9303b2c35b8be05a79b7014b4300b45bc734f3d818R97), or rather right before 
[line 124](https://github.com/openssl/openssl/pull/22185/files/8e0f3474abc5a53bc6304c6b3157a6b09c5cdfac#diff-db882b7e799eaa3257692a9303b2c35b8be05a79b7014b4300b45bc734f3d818R124) (which is closer to where it was before)?

Reply:
Before line 124 would be fine IMO.